### DISCUSSION TITLE
How to remove slider end misses from my skin/game or downvote change

### DISCUSSION THREAD

**Discussion by AhmadRehan98** [ppy/osu#26101] [Q&A] [answered]
This is the change I'm refering to: https://github.com/ppy/osu/pull/25951

These new small miss indicators are very distracting to me.
They happen a lot without a combo break, but it looks like one since it's an x.

Is there a way to disable/change them in my game or per skin? 

Also in the pull request peppy says it's out "for testing".
Where do I voice my dislike of this feature? Should I make a poll myself or is there another place for this?

**Comment by peppy** [comment, marked as answer]
There will be support for changing them in the near future.
